debian/control

```diff
@@ -3,9 +3,9 @@ Section: contrib/python
 Priority: extra
 Maintainer: Synapse Packaging team <packages@matrix.org>
 # keep this list in sync with the build dependencies in docker/Dockerfile-dhvirtualenv.
+# TODO: Remove the dependency on dh-systemd after dropping support for Ubuntu xenial
 Build-Depends:
- debhelper (>= 9),
- dh-systemd,
+ debhelper (>= 9.20160709) | dh-systemd,
```

Comment:
`dh-systemd` depends on `debhelper`, so if the newer `debhelper` is unavailable, we'll fall back to `dh-systemd` which will then itself pull in the older debhelper.

Nevertheless, we will only need to do this on Ubuntu xenial *without* xenial-backports, which should be exceptionally rare.

Reply:
> Nevertheless, we will only need to do this on Ubuntu xenial without xenial-backports, which should be exceptionally rare.

well, this is run in the build environment, which is a docker image we control. I don't think it has backports enabled, but either way, it's either 0% or 100% of occasions we run the build...

Reply:
> dh-systemd depends on debhelper, so if the newer debhelper is unavailable, we'll fall back to dh-systemd which will then itself pull in the older debhelper.

could you stick this comment in the file so that it doesn't get lost in the PR?

Reply:
> this is run in the build environment, which is a docker image we control. I don't think it has backports enabled, but either way, it's either 0% or 100% of occasions we run the build...

It does have backports enabled but we should specify these dependencies as robustly as possible in case someone needed to rebuild the package locally (e.g., `apt-get build-dep`, `apt-get source`, etc.)

> could you stick this comment

Done